knative-operator/hack/update-manifests.sh

```diff
@@ -37,8 +37,9 @@ function download_kafka {
 
 download_kafka knativekafka "$KNATIVE_EVENTING_KAFKA_VERSION" "${kafka_files[@]}"
 
-# Change the minavailable pdb for kafka-webhook to 1
-#git apply "$root/knative-operator/hack/007-eventing-kafka-pdb.patch"
+# Apply the patches
 
-# For 1.17 we still skip HPA/PDB
-git apply "$root/knative-operator/hack/001-remove_hpa_pdb.patch"
+for p in `ls $root/knative-operator/hack/*.patch|sort -n`
+do
+  git apply $p
+done
```

Comment:
I believe this is equivalent to `git apply $root/knative-operator/hack/*.patch` (see https://serverfault.com/a/122743 for globs generating a sorted list)

Reply:
I still like having explicit patch applies on our repo here, since that way we have a bit more context (expect for git blame)

Reply:
I believe this is, yes, but applying the patches in this way can give a clearer picture of which patch specifically failed, no?

Reply:
I switched back, I wanna get this PR out :D